package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vue2mapbox-gl",
-  "version": "0.13.1",
+  "version": "0.15.0",
```

Comment:
On npm it's version `0.14.0` already https://www.npmjs.com/package/vue2mapbox-gl